www_src/static/img/B.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
all these SVG have an aweful lot of "sketch" branding... I'd be highly in favour of stripping that away.